components/collator/src/provider.rs

```diff
@@ -557,4 +610,18 @@ impl CollationSpecialPrimaries<'_> {
         // See parse.cpp in genrb and getLastPrimaryForGroup in ICU4C.
         (u32::from(self.last_primaries.get(max_variable as usize).unwrap()) << 16) - 1
     }
+
+    #[allow(dead_code)]
+    pub(crate) fn is_compressible(&self, b: u8) -> bool {
+        // Unwrap OK by construction and pasting this
+        // into Compiler Explorer shows that the panic
+        // is optimized away.
```

Comment:
Is the Compiler Explorer remark still true? That is, is the math on `len()` really visible enough to the optimizer?